djangocms_bootstrap4/contrib/bootstrap4_link/forms.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from django.utils.safestring import mark_safe
+
+from djangocms_link.forms import LinkForm
+from djangocms_bootstrap4.constants import DEVICE_SIZES
+
+from .models import Bootstrap4Link
+
+
+class HorizontalRadioRenderer(forms.RadioSelect.renderer):
+
+    def render(self):
+        return mark_safe('\n'.join(['%s\n' % w for w in self]))
```

Comment:
`'\n'.join(iterable)` already adds a newline after each entry, do you need a second one?
if not then you can do return mark_safe('\n'.join(self))